.gitignore

```diff
@@ -0,0 +1,6 @@
+
```

Comment:
Did you intend to add this file?

Reply:
Add it to your  .git/info/exclude and be free of this issue forever.

Reply:
I add `.*` to `.gitignore` and then git ignores the gitignore